squema.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE animals ( name varchar(100) );

```

Comment:
- [ ] [OPTIONAL] Kindly remove this file `squema.sql` as this code is duplicated as you already created the `schema.sql` file.